main.js

```diff
@@ -23,12 +23,15 @@ function displayNumbers(allNumbers) {
         number.setAttribute('class', 'numberBox');
         document.querySelector('#play_area').appendChild(number);
     }
-
 }
 
 document.querySelector('#start').addEventListener('click', () => {
+    const randomNumbers = generateNumbers();
+
+    displayNumbers(randomNumbers);
+
     document.querySelector('#splash').style.display = 'none';
     document.querySelector('#game').style.display = 'block';
-    let randomNumbers = generateNumbers();
-    displayNumbers(randomNumbers);
+    document.querySelector('#game_timer').textContent = "Timer: 30";
```

Comment:
problem with this is that if the amount of time to play the game changes, ull have to remember to update it here.

how about we create a constant containing the total game time in timer.js, that you then use here, and you use it as the value to `timeLeft`. that way you only change it once to update both places